wsmaster/che-core-api-installer/src/main/java/org/eclipse/che/api/installer/server/model/impl/InstallerServerConfigImpl.java

```diff
@@ -0,0 +1,144 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2017 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.installer.server.model.impl;
+
+import com.google.common.base.Objects;
+
+import org.eclipse.che.api.core.model.workspace.config.ServerConfig;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author Anatolii Bazko
+ */
+@Entity(name = "InstallerServerConf")
+@Table(name = "installer_servers")
+public class InstallerServerConfigImpl implements ServerConfig {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "installer_id", nullable = false)
+    private String installerId;
```

Comment:
Are `installerId` and `installerVersion` usefull fields here or they are used only for mapping?
If only for mapping then maybe we can map `InstallerServerConfigImpl` and `InstallerImpl` in other way like map InstallerServerConfigImpl to InstalleImpl by InstallerServerConfigImpl#id field, then looks like `installerId` and `installerVersion` are redundant and you'll be able to create copy constructor for InstallerServerConfigImpl(ServerConfig). WDYT?